browser.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function () {
+  throw new Error('ws only supports Node.js! Please consider to use `isomorphic-ws` for both Node.js & Browser support! (See issue #1344 at https://github.com/websockets/ws/issues/1344 for more information.');
```

Comment:
Please keep the limit of 80 chars per line. Also, I don't want to endorse any particular wrapper, we are already doing  it in the README and there is no need to point to that particular issue, there a few already which are all duplicates.

I'd just use a very simple message like "ws does not work in the browser. Browser clients must use the native `WebSocket` object". I think this is sufficient to address the user into the right direction.